### ISSUE TITLE
Is there any way to customize the color and radius of the clustered circles?

### ISSUE THREAD

**Issue by axelerate** [SINTEF-9012/PruneCluster#151]
I would like to style the clusters so as to impart rules on how wide the cluster zones should be and what color pattern to abide to.

**Comment by yellowiscool** [comment]
Yes it's possible, but there is no real API for that. You have to do write some code.

[Check this source code for example.](https://github.com/SINTEF-9012/PruneCluster/blob/master/examples/random.10000-categories.html#L33-L108)
